### ISSUE TITLE
Unable to build on Xcode 12 due to PINCache dependency

### ISSUE THREAD

**Issue by joshavant** [Giphy/giphy-ios-sdk-ui-example#97]
Hello,

I'm unable to build on Xcode 12 due to Giphy's dependency on a version of PINCache which does not build on Xcode 12. I get the following build errors:
<img width="562" alt="Screen Shot 2020-09-16 at 10 15 47 AM" src="https://user-images.githubusercontent.com/830519/93357131-98c82580-f805-11ea-85ee-367cfb2c55e7.png">

My Podfile includes the Giphy pod in my project using the following declaration:
```
pod 'Giphy', '~> 2.0.2'
```

My Podfile.lock describes the actually installed version like so:
```
  - Giphy (2.0.2):
    - DeepDiff
    - libwebp
    - PINCache
```

My Podfile.lock describes the PINCache version installed from this dependency like so:
```
  - PINCache (3.0.1-beta.8):
    - PINCache/Arc-exception-safe (= 3.0.1-beta.8)
    - PINCache/Core (= 3.0.1-beta.8)
```

Looking on the PINCache project page, it appears the version following `3.0.1-beta.8` (which was dubbed `3.0.1`) added support for Xcode 12:
https://github.com/pinterest/PINCache/releases/tag/3.0.1

To fix this issue, It appears that Giphy needs to cut a new version with an updated PINCache dependency version set to `3.0.1`.

**Comment by cgmaier** [comment]
hey @joshavant that's odd - `2.0.2` should use `3.0.1`. I'm still investigating & figuring out next steps around some of the issues, but released a version of the SDK `2.0.3-Xcode-12` which uses the Xcode-12 command line tools for compilation, and successfully tested in Xcode 12 / iOS 14.

```
pod 'Giphy', :podspec => 'https://s3.amazonaws.com/sdk.mobile.giphy.com/SDK/2.0.3-Xcode-12/Giphy.podspec'
```

**Comment by cgmaier** [comment]
hey @joshavant I just updated the above pod spec - let me know if this helps!

```
rm -rf "${HOME}/Library/Caches/CocoaPods"
rm -rf "`pwd`/Pods/"
pod update
```

**Comment by joshavant** [comment]
I had to remove my `Podfile.lock`, too. But, yes, Cocoapods successfully downloaded that `.podspec`.

I'm having issues just getting a build to complete in Xcode 14 GM right now, so I can't confirm it successfully builds.

That said, looking at my Podfile.lock, I see that `PINCache 2.3` was the version that was resolved by Cocoapods to fulfill Giphy's dependency. So whatever was changed may not have fixed the problem.
